contracts/executable/AxelarGMPExecutable.sol

```diff
@@ -2,15 +2,16 @@
 
 pragma solidity ^0.8.0;
 
-import { IAxelarGMPGateway } from '../interfaces/IAxelarGMPGateway.sol';
+import { Pausable } from '../utils/Pausable.sol';
+import { IAxelarGMPGatewayWithToken } from '../interfaces/IAxelarGMPGatewayWithToken.sol';
 import { IAxelarGMPExecutable } from '../interfaces/IAxelarGMPExecutable.sol';
 
 /**
  * @title AxelarGMPExecutable
  * @dev Abstract contract to be inherited by contracts that need to execute cross-chain commands via Axelar's Gateway.
  * It implements the IAxelarGMPExecutable interface.
  */
-abstract contract AxelarGMPExecutable is IAxelarGMPExecutable {
+abstract contract AxelarGMPExecutable is IAxelarGMPExecutable, Pausable {
```

Comment:
App-specific functionality such as Pausable should not be added to these base contracts that any app using Axelar's GMP will use